android-sdk/src/main/java/org/sagebionetworks/bridge/android/BridgeConfig.java

```diff
@@ -48,7 +48,6 @@
  * </ul>
  */
 @AnyThread
-@Singleton
```

Comment:
How come you removed all the singleton annotations?

Reply:
They limited how reusable the component is. I'm using BridgeManagerProvider as an included `@Component` in BP, meaning I give an instance of it to the MPower `@Component`. These are still singletons in practice because they're scoped to the Application.

Dagger is just being extra safe/explicity because I could theoretically attach two different instances of these components and break the meaning of `@Singleton`